com.unity.cinemachine/Runtime/Behaviours/CinemachineSplineCart.cs

```diff
@@ -33,29 +34,39 @@ public enum UpdateMethods
         [Tooltip("When to move the cart, if Speed is non-zero")]
         public UpdateMethods UpdateMethod = UpdateMethods.Update;
 
-        /// <summary>How to interpret the Spline Position</summary>
-        [Tooltip("How to interpret the Spline Position.  If set to Knot, values are as follows: "
-            + "0 represents the first knot on the path, 1 is the second, and so on.  "
-            + "Values in-between are points on the spline in between the knots.  "
-            + "If set to Distance, then Spline Position represents distance along the spline.")]
-        public PathIndexUnit PositionUnits = PathIndexUnit.Distance;
```

Comment:
We can't change the API without a major bump 4.X.X

Reply:
Changed it to break less API